src/devtools/views/Components/TreeContext.js

```diff
@@ -369,16 +369,11 @@ function reduceSearchState(store: Store, state: State, action: Action): State {
       searchIndex = newSearchIndex;
     }
   }
-  if (didRequestSearch) {
-    if (searchIndex === null) {
-      selectedElementIndex = null;
-      selectedElementID = null;
```

Comment:
Effectively I just removed this branch.

Reply:
Yeah, [`?w=1`](https://github.com/bvaughn/react-devtools-experimental/pull/149/files?w=1) makes that clear 😁